src/assets/messages.json

```diff
@@ -2951,6 +2952,7 @@
     "sends-call-repeating-based-on-options": "Envoie cet appel sur une base répéter, selon les options à droite.",
     "sensitive-data": "Données sensibles",
     "separate-output-for-each-choice": "Sortie séparée pour chaque choix",
+    "advanced-configuration-of-outputs": "Configuration avancée des sorties",
```

Comment:
Thanks for fixing it. I also came across this in
https://github.com/FLOIP/flow-builder/commit/3224c0925cba88ef7e3a6a9722b784d404197cb1